pkg/aws/ec2/ec2.go

```diff
@@ -257,20 +241,19 @@ func (c *Client) GetInstances(ctx context.Context, vpcs ipamTypes.VirtualNetwork
 
 // describeVpcs lists all VPCs
 func (c *Client) describeVpcs(ctx context.Context) ([]ec2_types.Vpc, error) {
-	var vpcs []ec2_types.Vpc
-
-	c.limiter.Limit(ctx, "DescribeVpcs")
-
-	sinceStart := spanstat.Start()
-	output, err := c.ec2Client.DescribeVpcs(ctx, &ec2.DescribeVpcsInput{})
-	c.metricsAPI.ObserveAPICall("DescribeVpcs", deriveStatus(err), sinceStart.Seconds())
-	if err != nil {
-		return nil, err
+	result := []ec2_types.Vpc{}
```

Comment:
same here for the initialization of the slice.

Reply:
Updated